SPECS/libkcapi/libkcapi.spec

```diff
@@ -0,0 +1,567 @@
+
+
+%build
+%configure               \
+  --libdir=/%{_lib}      \
+  --disable-silent-rules \
+  --enable-kcapi-encapp  \
+  --enable-kcapi-dgstapp \
+  --enable-kcapi-hasher  \
+  --enable-kcapi-rngapp  \
+  --enable-kcapi-speed   \
+  --enable-kcapi-test    \
+  --enable-shared        \
+  --enable-static        \
+  --enable-sum-prefix=   \
+  --enable-sum-dir=/%{_lib} \
+  --with-pkgconfigdir=%{_libdir}/pkgconfig
+
+
+%install
+%make_install
+
+# Install sysctl.d preset.
+mkdir -p %{buildroot}%{_sysctldir}
+install -Dpm 0644 -t %{buildroot}%{_sysctldir} \
+  %{sysctl_prio}-%{name}-optmem_max.conf
+
+
+# Install replacement tools, if enabled.
+%if !%{with replace_coreutils}
+rm -f                            \
+  %{buildroot}%{_bindir}/md5sum       \
+  %{buildroot}%{_bindir}/sha*sum
+%endif
+
+%if !%{with replace_fipscheck}
+rm -f %{buildroot}%{_bindir}/fips*
+%endif
+
+%if !%{with replace_hmaccalc}
+rm -f %{buildroot}%{_bindir}/sha*hmac
+%endif
+
+# We don't ship autocrap dumplings.
+find %{buildroot} -type f -name "*.la" -delete -print
+
+# HMAC checksums are generated during __spec_install_post.
+%{_bindir}/find %{buildroot} -type f -name '*.hmac' -print -delete
+
+# Remove 0-size files.
+%{_bindir}/find %{buildroot} -type f -size 0 -print -delete
+
+
+%check
+
+# On some arches `/proc/sys/net/core/optmem_max` is lower than 20480,
+# which is the lowest limit needed to run the testsuite.  If that limit
+# is not met, we do not run it.
+%if %{test_optmem_max} >= 20480
+# Skip the testsuite on old kernels.
+%if %{lua:print(rpm.vercmp(posix.uname('%r'), '5.1'));} >= 0
+# Real testsuite.
+pushd test
+ENABLE_FUZZ_TEST=1 \
+NO_32BIT_TEST=1    \
+  ./test-invocation.sh
+popd
+%endif
+%endif
+
+
+%ldconfig_scriptlets
+
+
+%files
+%license COPYING*
+/%{_lib}/%{name}.so.%{vmajor}
+/%{_lib}/%{name}.so.%{version}
+/%{_lib}/fipscheck/%{name}.so.%{vmajor}.hmac
+/%{_lib}/fipscheck/%{name}.so.%{version}.hmac
+%if %{with_sysctl_tweak}
+%{_sysctldir}/%{sysctl_prio}-%{name}-optmem_max.conf
+%endif
+
+%files devel
+%{_includedir}/kcapi.h
+%{_mandir}/man3/kcapi_*.3.*
+/%{_lib}/%{name}.so
+%{_libdir}/pkgconfig/%{name}.pc
+
+%if %{with replace_coreutils}
+%files checksum
+%{_bindir}/md5sum
+%{_bindir}/sha*sum
+/%{_lib}/fipscheck/md5sum.hmac
+/%{_lib}/fipscheck/sha*sum.hmac
+%{endifhttps}://src.fedoraproject.org/rpms/libkcapi.gitreplace_hmaccalc}
+
+%files hmaccalc
+%{_bindir}/sha*hmac
+/%{_lib}/hmaccalc/sha*hmac.hmac
+%endif
+
+%if %{with replace_fipscheck}
+%files fipscheck
+%{_bindir}/fips*
+/%{_lib}/fipscheck/fips*.hmac
+%endif
+
+%if %{with replace_hmaccalc}
+%files hmaccalc
+%{_bindir}/sha*hmac
+/%{_lib}/hmaccalc/sha*hmac.hmac
+%endif
+
+%files static
+/%{_lib}/%{name}.a
+
+%files tools
+%{_bindir}/kcapi*
+%{_mandir}/man1/kcapi*.1.*
+
+%if %{with test_package}
+%files tests
+%{_libexecdir}/%{name}/*
+%endif
+
+
+%changelog
+* Tue Jan 19 2021 Nicolas Ontiveros <niontive@microsoft.com> - 1.2.0-4
+- Initiali CBL-Mariner import from Fedora 33.
```

Comment:
Please verify the license and reflect that in the changelog (we usually use the "License verified" phrase).